lib/codegen/meta-python/isa/x86/recipes.py

```diff
@@ -801,37 +803,50 @@ def valid_scale(iform):
 
 # XX /r register-indirect store with no offset.
 st = TailRecipe(
-        'st', Store, size=1, ins=(GPR, GPR_ZERO_DEREF_SAFE), outs=(),
+        'st', Store, base_size=1, ins=(GPR, GPR), outs=(),
         instp=IsEqual(Store.offset, 0),
         clobbers_flags=False,
+        compute_size="size_plus_maybe_offset_1",
```

Comment:
When I first read this, it wasn't obvious that the `_1` referred to `in_reg1`. Could we name it something like `size_plus_maybe_offset_for_in_reg1`?